src/components/atoms/input/Radio/Radio.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { HTMLAttributes, useState } from "react";
+
+export interface RadioProps extends HTMLAttributes<HTMLElement> {
+    id?: string;
+    className?: string;
+    value: string;
+    checked: boolean;
+    type: string;
```

Comment:
do you need this?

Reply:
I am not sure actually, let's chat about this. Still trying to find my way around Typescript :)